presto-docs/src/main/sphinx/functions/array.rst

```diff
@@ -96,12 +96,10 @@ Array Functions
     Subsets array ``x`` starting from index ``start`` (or starting from the end
     if ``start`` is negative) with a length of ``length``.
 
-.. function:: zip(array1, array2[, array3[, array4]]) -> array<row>
+.. function:: zip(array1, array2[, ...]) -> array<row>
```

Comment:
It is effectively a var args function, but is limited to a fixed arity due to limitations in the engine. We could easily increase it to 5, 10, etc.

The user reading the documentation can think of it as a var arg function. Other functions like `concat` also have limits (such as Java methods being limited to 255 arguments).